diff -Nur e2fsprogs-1.22.orig/doc/e2compr.texinfo e2fsprogs-1.22/doc/e2compr.texinfo
--- e2fsprogs-1.22.orig/doc/e2compr.texinfo	Sat Aug  4 19:09:19 2001
+++ e2fsprogs-1.22/doc/e2compr.texinfo	Sat Aug  4 19:11:36 2001
@@ -24,9 +24,8 @@
 @c %**end of header
 
 @c What to put in info directory (e.g. /usr/info/dir).
-@dircategory Miscellaneous
+@dircategory Miscellaneous:
 @direntry
-* e2compr::				User manual for ext2 compression.
 * e2bitmap: (e2compr)e2bitmap.		Display which clusters are compressed.
 * e2ratio: (e2compr)e2ratio.		Ext2 compression statistics.
 @end direntry
