--- e2fsprogs-1.42.12/doc/e2compr.texinfo.orig	2014-10-24 16:56:18.270058857 +0200
+++ e2fsprogs-1.42.12/doc/e2compr.texinfo	2014-10-24 18:30:24.926488556 +0200
@@ -24,11 +24,10 @@
 @c %**end of header
 
 @c What to put in info directory (e.g. /usr/info/dir).
-@dircategory Miscellaneous
+@dircategory Miscellaneous:
 @direntry
-* e2compr::				User manual for ext2 compression.
-* e2bitmap: (e2compr)e2bitmap.		Display which clusters are compressed.
-* e2ratio: (e2compr)e2ratio.		Ext2 compression statistics.
+* e2bitmap: (e2compr)e2bitmap.		Display which clusters are compressed
+* e2ratio: (e2compr)e2ratio.		Ext2 compression statistics
 @end direntry
 @c :vers: Grab the stuff about e2compress from an old manual,
 @c once it is ready for 0.4.  Also get the direntry stuff for it.
@@ -986,8 +985,7 @@
 @c Should be both unindented and in fixed-width font.
 @c How do I do this?
 @c `@format' is suitable for info.
-@t{
-@format
+@verbatim
  ch+               du               tar (read)        ch-              cs alg  
 ~~~~~~~~~~~~~~~~~ ~~~~~~~~~~~~~~~~ ~~~~~~~~~~~~~~~~~ ~~~~~~~~~~~~~~~~~ ~~ ~~~~~
 0.16u 203.9s 3:51 2257 0.16u 0.84s 0.59u 93.88s 1:49 0.17u 20.09s 0:36 32 gzip9
@@ -1039,8 +1037,7 @@
 0.19u  32.3s 1:10 3965 0.16u 0.71s 0.64u 17.71s 0:40 0.21u 16.35s 0:58 04 lzrw3a
 
 0.17u  19.3s 1:00 4025 0.11u 0.74s 0.66u 10.36s 0:34 0.26u  9.33s 0:45 04 lzv1
-@end format
-}
+@end verbatim
 
 @c ------------------------------------------------------------------
 @node none, future methods, Compression choices, Algorithms
@@ -1340,7 +1337,7 @@
 are) tend to have a high turnover of files (deleting files to make way
 for new files, which have to be written in the cracks occupied by the
 files just deleted), which causes high fragmentation.
-@xref{defragmenter} for comments on using a defragmenter.
+For comments on using a defragmenter, @pxref{defragmenter}.
 
 
 @c ------------------------------------------------------------------
