diff -ur e2fsprogs-1.41.3/configure.in e2fsprogs-1.41.3-diet/configure.in
--- e2fsprogs-1.41.3/configure.in	2009-03-17 22:03:38.000000000 +0100
+++ e2fsprogs-1.41.3-diet/configure.in	2009-03-17 22:01:22.000000000 +0100
@@ -701,7 +701,7 @@
 	[#include <sys/types.h>
 	 #include <sys/socket.h>])
 dnl
-AC_CHECK_FUNCS(chflags getrusage llseek lseek64 open64 fstat64 ftruncate64 getmntinfo strtoull strcasecmp srandom jrand48 fchown mallinfo fdatasync strnlen strptime strdup sysconf pathconf posix_memalign memalign valloc __secure_getenv prctl mmap utime setresuid setresgid usleep nanosleep getdtablesize getrlimit)
+AC_CHECK_FUNCS(chflags getrusage llseek lseek64 open64 fstat64 stat64 ftruncate64 getmntinfo strtoull strcasecmp srandom jrand48 fchown mallinfo fdatasync strnlen strptime strdup sysconf pathconf posix_memalign memalign valloc __secure_getenv prctl mmap utime setresuid setresgid usleep nanosleep getdtablesize getrlimit)
 dnl
 dnl Check to see if -lsocket is required (solaris) to make something
 dnl that uses socket() to compile; this is needed for the UUID library
diff -ur e2fsprogs-1.41.3/misc/util.c e2fsprogs-1.41.3-diet/misc/util.c
--- e2fsprogs-1.41.3/misc/util.c	2008-10-07 16:22:39.000000000 +0200
+++ e2fsprogs-1.41.3-diet/misc/util.c	2009-03-17 22:01:31.000000000 +0100
@@ -79,7 +79,7 @@
 void check_plausibility(const char *device)
 {
 	int val;
-#ifdef HAVE_OPEN64
+#ifdef HAVE_STAT64
 	struct stat64 s;
 
 	val = stat64(device, &s);
--- e2fsprogs-1.41.5/MCONFIG.in~	2009-04-22 13:49:25.000000000 +0200
+++ e2fsprogs-1.41.5/MCONFIG.in	2009-05-20 15:22:35.879974273 +0200
@@ -79,7 +79,7 @@
 LIBINTL = @LIBINTL@
 DEPLIBSS = $(LIB)/libss@LIB_EXT@
 DEPLIBUUID = $(LIB)/libuuid@LIB_EXT@
-DEPLIBBLKID = @DEPLIBBLKID@ @PRIVATE_LIBS_CMT@ $(LIBUUID)
+DEPLIBBLKID = @DEPLIBBLKID@ @PRIVATE_LIBS_CMT@ $(LIB)/libuuid@LIB_EXT@
 
 STATIC_LIBSS = $(LIB)/libss@STATIC_LIB_EXT@ @DLOPEN_LIB@
 STATIC_LIBCOM_ERR = $(LIB)/libcom_err@STATIC_LIB_EXT@ @SEM_INIT_LIB@
@@ -89,7 +89,7 @@
 STATIC_LIBBLKID = @STATIC_LIBBLKID@ $(STATIC_LIBUUID)
 DEPSTATIC_LIBSS = $(LIB)/libss@STATIC_LIB_EXT@
 DEPSTATIC_LIBUUID = $(LIB)/libuuid@STATIC_LIB_EXT@
-DEPSTATIC_LIBBLKID = @DEPSTATIC_LIBBLKID@ $(STATIC_LIBUUID)
+DEPSTATIC_LIBBLKID = @DEPSTATIC_LIBBLKID@ $(LIB)/libuuid@STATIC_LIB_EXT@
 
 PROFILED_LIBSS = $(LIB)/libss@PROFILED_LIB_EXT@ @DLOPEN_LIB@
 PROFILED_LIBCOM_ERR = $(LIB)/libcom_err@PROFILED_LIB_EXT@ @SEM_INIT_LIB@
@@ -98,7 +98,7 @@
 PROFILED_LIBUUID = $(LIB)/libuuid@PROFILED_LIB_EXT@ @SOCKET_LIB@ 
 PROFILED_LIBBLKID = @PROFILED_LIBBLKID@ $(PROFILED_LIBUUID)
 DEPPROFILED_LIBUUID = $(LIB)/libuuid@PROFILED_LIB_EXT@
-DEPPROFILED_LIBBLKID = @PROFILED_LIBBLKID@ $(PROFILED_LIBUUID)
+DEPPROFILED_LIBBLKID = @PROFILED_LIBBLKID@ $(LIB)/libuuid@PROFILED_LIB_EXT@
 
 # An include directive pointing to a directory holding enough linux-like
 # include files to satisfy some programs here
