--- e2fsprogs-1.33/e2fsck/message.c.orig	2002-09-30 17:03:32.000000000 +0200
+++ e2fsprogs-1.33/e2fsck/message.c	2003-04-26 21:45:30.000000000 +0200
@@ -218,12 +218,12 @@
 			break;
 	}
 	if (*cpp) {
-		str = (*cpp) + 1;
+		str = _(*cpp) + 1;
 		if (*first && islower(*str)) {
 			*first = 0;
 			fputc(toupper(*str++), stdout);
 		}
-		print_e2fsck_message(ctx, _(str), pctx, *first);
+		print_e2fsck_message(ctx, str, pctx, *first);
 	} else
 		printf("@%c", ch);
 }
