--- e2fsprogs-1.40.5/configure.in~	2008-02-04 19:04:56.000000000 +0200
+++ e2fsprogs-1.40.5/configure.in	2008-02-04 19:05:18.956729223 +0200
@@ -815,22 +815,6 @@
 	root_libdir='${root_prefix}/lib'
 	root_sysconfdir='${root_prefix}/etc'
 fi
-if test "$bindir" != '${exec_prefix}/bin'; then
-    root_bindir=$bindir
-    echo "Setting root_bindir to $root_bindir"
-fi
-if test "$sbindir" != '${exec_prefix}/sbin'; then
-    root_sbindir=$sbindir
-    echo "Setting root_sbindir to $root_sbindir"
-fi
-if test "$libdir" != '${exec_prefix}/lib'; then
-    root_libdir=$libdir
-    echo "Setting root_libdir to $root_libdir"
-fi
-if test "$sysconfdir" != '${prefix}/etc'; then
-    root_sysconfdir=$sysconfdir
-    echo "Setting root_sysconfdir to $root_sysconfdir"
-fi
 AC_SUBST(root_prefix)
 AC_SUBST(root_bindir)
 AC_SUBST(root_sbindir)
