--- e2fsprogs-1.27/lib/ext2fs/unix_io.c.orig	Sun Feb 24 22:04:00 2002
+++ e2fsprogs-1.27/lib/ext2fs/unix_io.c	Fri Apr 12 17:58:53 2002
@@ -350,6 +350,10 @@
 	 * Work around a bug in 2.4.10+ kernels where writes to block
 	 * devices are wrongly getting hit by the filesize limit.
 	 */
+	/* disabled because causes more problems (which can't be solved
+	 * using ulimit) when using glibc compiled on 2.2.x kernel headers
+	 */
+#if 0
 	if ((flags & IO_FLAG_RW) && 
 	    (fstat(data->dev, &st) == 0) &&
 	    (S_ISBLK(st.st_mode))) {
@@ -365,6 +369,7 @@
 		}
 	}
 #endif
+#endif
 	*channel = io;
 	return 0;
 
