--- e2fsprogs-1.43/configure.ac.orig	2008-09-01 17:36:59.000000000 +0200
+++ e2fsprogs-1.43/configure.ac	2008-09-11 16:50:13.136992337 +0200
@@ -801,22 +801,6 @@
 	root_libdir='${root_prefix}/lib'
 	root_sysconfdir='${root_prefix}/etc'
 fi
-if test "$bindir" != '${exec_prefix}/bin'; then
-    root_bindir=$bindir
-    AC_MSG_RESULT([Setting root_bindir to $root_bindir])
-fi
-if test "$sbindir" != '${exec_prefix}/sbin'; then
-    root_sbindir=$sbindir
-    AC_MSG_RESULT([Setting root_sbindir to $root_sbindir])
-fi
-if test "$libdir" != '${exec_prefix}/lib'; then
-    root_libdir=$libdir
-    AC_MSG_RESULT([Setting root_libdir to $root_libdir])
-fi
-if test "$sysconfdir" != '${prefix}/etc'; then
-    root_sysconfdir=$sysconfdir
-    AC_MSG_RESULT([Setting root_sysconfdir to $root_sysconfdir])
-fi
 AC_SUBST(root_prefix)
 AC_SUBST(root_bindir)
 AC_SUBST(root_sbindir)
