--- e2fsprogs-1.21/doc/libext2fs.texinfo.orig	Sat Jun 16 00:53:48 2001
+++ e2fsprogs-1.21/doc/libext2fs.texinfo	Sun Jun 17 19:56:33 2001
@@ -6,11 +6,11 @@
 @comment %**end of header
 
 @ifinfo
-@dircategory Development
 @format
-START-INFO-DIR-ENTRY
-* libext2fs: (libext2fs.info).   The EXT2FS library.
-END-INFO-DIR-ENTRY
+@dircategory Libraries:
+@direntry
+* libext2fs: (libext2fs)		The EXT2FS library
+@end direntry
 @end format
 @end ifinfo
 
