--- e2fsprogs-1.45.5/doc/libext2fs.texinfo.orig	2020-01-07 00:10:17.000000000 +0100
+++ e2fsprogs-1.45.5/doc/libext2fs.texinfo	2020-02-08 21:19:46.530578543 +0100
@@ -6,9 +6,9 @@
 @comment %**end of header
 
 @ifinfo
-@dircategory Development
+@dircategory Libraries:
 @direntry
-* libext2fs: (libext2fs).                  The EXT2FS library.
+* libext2fs: (libext2fs).		The EXT2FS library
 @end direntry
 @end ifinfo
 
