diff -Nur --exclude *~ e2fsprogs-1.19/e2fsck/Makefile.in e2fsprogs-1.19.new/e2fsck/Makefile.in
--- e2fsprogs-1.19/e2fsck/Makefile.in	Fri Jul  7 06:35:44 2000
+++ e2fsprogs-1.19.new/e2fsck/Makefile.in	Mon May 14 02:33:43 2001
@@ -95,7 +95,7 @@
 		$(STATIC_LIBS) 
 
 e2fsck.shared: $(OBJS)  $(DEPLIBS)
-	$(LD) $(ALL_LDFLAGS) -o e2fsck.shared $(OBJS) $(LIBS) 
+	$(LD) $(ALL_LDFLAGS) -o e2fsck.shared $(OBJS) $(LIBS) $(LDLIBS)
 
 e2fsck.profiled: $(PROFILED_OBJS)  $(PROFILED_DEPLIBS)
 	$(LD) $(ALL_LDFLAGS) -g -pg -o e2fsck.profiled $(PROFILED_OBJS) \
diff -Nur --exclude *~ e2fsprogs-1.19/misc/Makefile.in e2fsprogs-1.19.new/misc/Makefile.in
--- e2fsprogs-1.19/misc/Makefile.in	Tue Jan 18 17:30:26 2000
+++ e2fsprogs-1.19.new/misc/Makefile.in	Mon May 14 02:32:28 2001
@@ -49,44 +49,44 @@
 all:: $(SPROGS) $(UPROGS) $(USPROGS) $(SMANPAGES) $(UMANPAGES)
 
 findsuper: findsuper.o
-	$(CC) $(ALL_LDFLAGS) -o findsuper findsuper.o
+	$(CC) $(ALL_LDFLAGS) -o findsuper findsuper.o $(LDLIBS)
 
-tune2fs: $(TUNE2FS_OBJS) $(DEPLIBS_E2P) $(DEPLIBUUID)
-	$(CC) $(ALL_LDFLAGS) -o tune2fs $(TUNE2FS_OBJS) $(LIBS_E2P) $(LIBUUID)
+tune2fs: $(TUNE2FS_OBJS) $(DEPLIBS_E2P) $(DEPLIBUUID) 
+	$(CC) $(ALL_LDFLAGS) -o tune2fs $(TUNE2FS_OBJS) $(LIBS_E2P) $(LIBUUID) $(LDLIBS)
 
 e2label: $(E2LABEL_OBJS) 
-	$(CC) $(ALL_LDFLAGS) -o e2label $(E2LABEL_OBJS) 
+	$(CC) $(ALL_LDFLAGS) -o e2label $(E2LABEL_OBJS) $(LDLIBS)
 
 mklost+found: $(MKLPF_OBJS)
-	$(CC) $(ALL_LDFLAGS) -o mklost+found $(MKLPF_OBJS)
+	$(CC) $(ALL_LDFLAGS) -o mklost+found $(MKLPF_OBJS) $(LDLIBS)
 
 mke2fs: $(MKE2FS_OBJS) $(DEPLIBS) $(LIBE2P) $(DEPLIBUUID)
-	$(CC) $(ALL_LDFLAGS) -o mke2fs $(MKE2FS_OBJS) $(LIBS) $(LIBE2P) $(LIBUUID)
+	$(CC) $(ALL_LDFLAGS) -o mke2fs $(MKE2FS_OBJS) $(LIBS) $(LIBE2P) $(LIBUUID) $(LDLIBS)
 
 mke2fs.static: $(MKE2FS_OBJS) $(STATIC_DEPLIBS) $(STATIC_LIBE2P) $(DEPSTATIC_LIBUUID)
 	$(CC) $(ALL_LDFLAGS) -static -o mke2fs.static $(MKE2FS_OBJS) \
-		$(STATIC_LIBS) $(STATIC_LIBE2P) $(STATIC_LIBUUID)
+		$(STATIC_LIBS) $(STATIC_LIBE2P) $(STATIC_LIBUUID) $(LDLIBS)
 
 resize2fs: resize2fs.o $(DEPLIBS)
-	$(CC) $(ALL_LDFLAGS) -o resize2fs resize2fs.o $(LIBS)
+	$(CC) $(ALL_LDFLAGS) -o resize2fs resize2fs.o $(LIBS) $(LDLIBS)
 
 chattr: $(CHATTR_OBJS) $(DEPLIBS_E2P)
-	$(CC) $(ALL_LDFLAGS) -o chattr $(CHATTR_OBJS) $(LIBS_E2P)
+	$(CC) $(ALL_LDFLAGS) -o chattr $(CHATTR_OBJS) $(LIBS_E2P) $(LDLIBS)
 
 lsattr: $(LSATTR_OBJS) $(DEPLIBS_E2P)
-	$(CC) $(ALL_LDFLAGS) -o lsattr $(LSATTR_OBJS) $(LIBS_E2P)
+	$(CC) $(ALL_LDFLAGS) -o lsattr $(LSATTR_OBJS) $(LIBS_E2P) $(LDLIBS)
 
 uuidgen: $(UUIDGEN_OBJS) $(DEPLIBUUID)
-	$(CC) $(ALL_LDFLAGS) -o uuidgen $(UUIDGEN_OBJS) $(LIBUUID)
+	$(CC) $(ALL_LDFLAGS) -o uuidgen $(UUIDGEN_OBJS) $(LIBUUID) $(LDLIBS)
 
-dumpe2fs: $(DUMPE2FS_OBJS) $(DEPLIBS_E2P)
+dumpe2fs: $(DUMPE2FS_OBJS) $(DEPLIBS_E2P) $(LDLIBS)
 	$(CC) $(ALL_LDFLAGS) -o dumpe2fs $(DUMPE2FS_OBJS) $(LIBS_E2P)
 
 fsck: $(FSCK_OBJS)
-	$(CC) $(ALL_LDFLAGS) -o fsck $(FSCK_OBJS)
+	$(CC) $(ALL_LDFLAGS) -o fsck $(FSCK_OBJS) $(LDLIBS)
 
 badblocks: $(BADBLOCKS_OBJS) $(DEPLIBS)
-	$(CC) $(ALL_LDFLAGS) -o badblocks $(BADBLOCKS_OBJS) $(LIBS)
+	$(CC) $(ALL_LDFLAGS) -o badblocks $(BADBLOCKS_OBJS) $(LIBS) $(LDLIBS)
 
 tune2fs.8: $(DEP_SUBSTITUTE) $(srcdir)/tune2fs.8.in
 	$(SUBSTITUTE) $(srcdir)/tune2fs.8.in tune2fs.8
diff -Nur --exclude *~ e2fsprogs-1.19/util/Makefile.in e2fsprogs-1.19.new/util/Makefile.in
--- e2fsprogs-1.19/util/Makefile.in	Fri Nov 19 19:42:30 1999
+++ e2fsprogs-1.19.new/util/Makefile.in	Mon May 14 02:56:26 2001
@@ -21,7 +21,7 @@
 all:: $(PROGS)
 
 subst: subst.o
-	$(BUILD_CC) $(ALL_LDFLAGS) -o subst subst.o
+	$(BUILD_CC) $(ALL_LDFLAGS) -o subst subst.o $(LDLIBS)
 
 clean:
 	$(RM) -f $(PROGS) \#* *.s *.o *.a *~ core
