--- e2fsprogs-1.45.3/configure.ac.orig	2019-07-16 18:18:01.489838520 +0200
+++ e2fsprogs-1.45.3/configure.ac	2019-07-16 18:55:50.620878916 +0200
@@ -872,7 +872,7 @@
 AC_SUBST(PACKAGE)
 AC_SUBST(VERSION)
 
-AM_GNU_GETTEXT
+AM_GNU_GETTEXT([external])
 dnl
 dnl End of configuration options
 dnl
@@ -1581,9 +1581,6 @@
 if test -n "$CPPFLAGS" ; then
 	INCLUDES="$INCLUDES $CPPFLAGS"
 fi
-if test "$USE_INCLUDED_LIBINTL" = "yes" ; then
-	INCLUDES=$INCLUDES' -I$(top_builddir)/intl -I$(top_srcdir)/intl'
-fi
 if test -n "$WITH_DIET_LIBC" ; then
 	INCLUDES="$INCLUDES -D_REENTRANT"
 fi
--- e2fsprogs-1.45.3/Makefile.in.orig	2019-07-15 03:03:14.000000000 +0200
+++ e2fsprogs-1.45.3/Makefile.in	2019-07-16 18:58:17.380083853 +0200
@@ -19,7 +19,7 @@
 @ALL_CMT@EXT2FS_LIB_SUBDIR= lib/ext2fs
 
 LIB_SUBDIRS=lib/et lib/ss $(E2P_LIB_SUBDIR) $(UUID_LIB_SUBDIR) \
-	$(BLKID_LIB_SUBDIR) $(SUPPORT_LIB_SUBDIR) $(EXT2FS_LIB_SUBDIR) intl
+	$(BLKID_LIB_SUBDIR) $(SUPPORT_LIB_SUBDIR) $(EXT2FS_LIB_SUBDIR)
 
 PROG_SUBDIRS=e2fsck $(DEBUGFS_DIR) misc $(RESIZE_DIR) tests/progs po \
 	$(E2SCRUB_DIR)
