--- e2fsprogs-1.34/lib/et/et_h.awk.wiget	2003-10-09 00:40:04.000000000 +0200
+++ e2fsprogs-1.34/lib/et/et_h.awk	2003-10-09 00:44:40.000000000 +0200
@@ -150,6 +150,10 @@
 	prefix_str = $2 "_"
 }
 
+/^[ \t]*(prefix)[ \t]*$/ {
+	prefix_str = ""
+}
+
 /^[ \t]*(index)[ \t]+[A-Z_0-9]+/ {
 	new_idx = $2
 	curr_low += curr_sign * (new_idx - curr_idx)
--- e2fsprogs-1.34/lib/et/et_c.awk.wiget	2003-10-09 00:45:08.000000000 +0200
+++ e2fsprogs-1.34/lib/et/et_c.awk	2003-10-09 00:45:35.000000000 +0200
@@ -184,6 +184,10 @@
 	prefix_str = $2 "_"
 }
 
+/^[ \t]*(prefix)[ \t]*$/ {
+	prefix_str = ""
+}
+
 /^[ \t]*(index)[ \t]+[A-Z_0-9]+/ {
 	new_idx = $2
 	for (i = table_item_count ; i < new_idx; i++) {
