--- e2fsprogs-1.32/e2fsck/message.c.orig	Wed Jun 26 05:26:47 2002
+++ e2fsprogs-1.32/e2fsck/message.c	Tue Nov 12 19:03:29 2002
@@ -210,15 +210,16 @@
 					  struct problem_context *pctx,
 					  int *first)
 {
-	const char **cpp, *str;
+	const char **cpp, *str, *ncpp;
 	
 	/* Search for the abbreviation */
 	for (cpp = abbrevs; *cpp; cpp++) {
-		if (ch == *cpp[0])
+		ncpp = gettext(*cpp);
+		if (ch == ncpp[0])
 			break;
 	}
 	if (*cpp) {
-		str = (*cpp) + 1;
+		str = ncpp + 1;
 		if (*first && islower(*str)) {
 			*first = 0;
 			fputc(toupper(*str++), stdout);
