--- e2fsprogs-1.40.5/lib/blkid/probe.c~	2008-02-04 19:45:37.000000000 +0200
+++ e2fsprogs-1.40.5/lib/blkid/probe.c	2008-02-04 19:45:53.367093916 +0200
@@ -779,8 +779,10 @@
 		   NSR01 for UDF 1.00
 		   NSR02 for UDF 1.50
 		   NSR03 for UDF 2.00 */
-		if (!memcmp(isosb->vd_id, "NSR0", 4))
-			return 0;
+		if (!memcmp(isosb->vd_id, "NSR0", 4)) {
+			return probe_iso9660(probe, id, buf);
+		}
+
 		for (m = udf_magic; *m; m++)
 			if (!memcmp(*m, isosb->vd_id, 5))
 				break;
