diff -durN -x '*~' e2fsprogs-1.34.orig/misc/util.c e2fsprogs-1.34/misc/util.c
--- e2fsprogs-1.34.orig/misc/util.c	2004-12-19 01:11:23.620322096 +0100
+++ e2fsprogs-1.34/misc/util.c	2004-12-19 01:12:32.557842000 +0100
@@ -32,6 +32,10 @@
 #include "blkid/blkid.h"
 #include "util.h"
 
+#ifndef SCSI_DISK_MAJOR
+#define SCSI_DISK_MAJOR SCSI_DISK0_MAJOR
+#endif
+
 #ifndef HAVE_STRCASECMP
 int strcasecmp (char *s1, char *s2)
 {
